### PR TITLE
Nexus time plus card

### PR DESCRIPTION
**Pull request by Dan-work1**

### CODE REVIEW

File: ResSimpy/Nexus/nexus_collect_tables.py [hunk 1]
```diff
@@ -60,9 +62,33 @@ def collect_all_tables_to_objects(nexus_file: File, table_object_map: dict[str,
...
+                current_date = time_value
+            else:
+                plus_value = get_expected_token_value(token='PLUS', token_line=line, file_list=file_as_list,
+                                                      custom_message="Cannot find the date associated with the TIME "
+                                                                     f"PLUS card in {line=} at line number {index}")
+
+                if current_date is None:
+                    raise ValueError("Cannot calculate PLUS date without access to the initial date.")
+                else:
+                    "Cannot calculate PLUS date without access to the initial date."
```
Comment (jonnyh404) [inline ResSimpy/Nexus/nexus_collect_tables.py:78]: Is this supposed to be a comment or is it a rogue string from above?

**Review by jonnyh404 — APPROVED** [review]
LGTM - see 1 comment for clean up of a rogue string

File: ResSimpy/Nexus/nexus_collect_tables.py (hunk 1 above)
Comment (Dan-work1) [reply ResSimpy/Nexus/nexus_collect_tables.py:78]: Yes that's not supposed to be there. I'll remove it, thanks.

**Review by cflynn3 — APPROVED** [review]
Looks good to me.

**Pull request merged**